index.php

```diff
@@ -93,6 +93,19 @@
                         <td class="task__date"></td>
                     </tr>
                     <!--показывать следующий тег <tr/>, если переменная $show_complete_tasks равна единице-->
+                    <?php if ($show_complete_tasks == 1) : ?>
```

Comment:
```suggestion
                    <?php if ($show_complete_tasks === 1) : ?>
```
Правильнее выполнять строгие проверки, что бы избежать неявного приведения типов